### PR TITLE
fix windows utf8 encoding 2/3

### PR DESCRIPTION
**Pull request by phaustin**
second of three parts -- see https://github.com/executablebooks/jupyter-book/pull/719

### CODE REVIEW

File: tests/notebooks/external_output.ipynb [hunk 1]
```diff
@@ -11,29 +11,27 @@
...
+      "metadata": {},
+      "outputs": [],
+      "source": []
     }
   ],
   "metadata": {
     "test_name": "notebook1",
     "kernelspec": {
-      "display_name": "Python 3",
+      "display_name": "Python 3.7.6 64-bit ('wintest': conda)",
```
Comment (chrisjsewell) [inline tests/notebooks/external_output.ipynb:32]:
```suggestion
      "display_name": "Python 3",
```

File: tests/notebooks/external_output.ipynb [hunk 2]
```diff
@@ -11,29 +11,27 @@
...
       "language": "python",
-      "name": "python3"
+      "name": "python37664bitwintestconda98645fcb790a40ef81b350c15db9f1ca"
```
Comment (chrisjsewell) [inline tests/notebooks/external_output.ipynb:34]:
```suggestion
      "name": "python3"
```

**Comment by choldgraf** [comment]
what's the reason for doing this in three parts? I'm a bit confused...

File: tests/notebooks/external_output.ipynb (hunk 1 above)
Comment (phaustin) [reply tests/notebooks/external_output.ipynb]: fixed

File: tests/notebooks/external_output.ipynb (hunk 2 above)
Comment (phaustin) [reply tests/notebooks/external_output.ipynb]: fixed

**Comment by phaustin** [comment]
the names were just to indicate that all three repositories need to be updated before the encoding
test will pass

**Comment by choldgraf** [comment]
ah gotcha OK - so we should get each PR into a state where we're ready to merge, and then do it all at once yeah?

**Comment by phaustin** [comment]
mainly just need all three before  we remove item 1 in the [windows instructions](https://github.com/phaustin/jupyter-book/blob/windows/docs/advanced/advanced.md#working-on-windows)

**Comment by choldgraf** [comment]
sounds good - well this one looks good and we have an approval, so I'm merging!

**Pull request merged**